pkg/cmd/kafka/update/update.go

```diff
@@ -213,9 +209,14 @@ func run(opts *options) error {
 }
 
 func runInteractivePrompt(opts *options, kafkaInstance *kafkamgmtclient.KafkaRequest) (err error) {
-	opts.owner, err = selectOwnerInteractive(opts.Context, opts)
-	if err != nil {
-		return err
+
+	if opts.userIsOrgAdmin {
+		opts.owner, err = selectOwnerInteractive(opts.Context, opts)
+		if err != nil {
+			return err
+		}
+	} else {
+		opts.owner = kafkaInstance.GetOwner()
```

Comment:
I think we should askForNewOwnerName - as opposed to select for admin.
Current logic does assign the same owner.

Reply:
as in enter the name directly as the select is for admins only?

Reply:
Only admins can fetch users but anyone can put username that they know and should be able to give their kafka away.